src/editor/components/filter/QueryEditorFieldAndOperator.tsx

```diff
@@ -121,9 +129,10 @@ export const QueryEditorFieldAndOperator: React.FC<Props> = (props: Props) => {
       <QueryEditorField
         value={value?.property}
         fields={fields}
-        templateVariableOptions={templateVariableOptions}
+        templateVariableOptions={props.templateVariableOptions}
         onChange={onFieldChanged}
         placeholder="Choose column..."
+        aria-label="choose column"
```

Comment:
This `aria-label` will be read to screen readers users possibly twice (some screen readers will read out the placeholder too) without context (the `Where (filter)` does not have an aria-label) so not very accessible.

* From an a11y perspective, using an aria-label in the label and a labelled-by here would be the preferred option
* For testing we could use an `id` here

Alternatively, as an easy but less best-practicy way we could have the `aria-label` here more descriptive (e.g. `choose column for where filter` )?

Reply:
the underlying AsyncSelect is tricky to update so I'll go with updating the aria label